config/sidekiq_scheduler.yml

```diff
@@ -176,10 +176,10 @@ VAForms::FetchLatest:
   class: VAForms::FormReloader
   description: "Fetches latest VA Forms"
 
-AppsApi::LoadApplications:
-  in: '1m'
-  class: AppsApi::DirectoryLoader
```

Comment:
Can you tell me more about the removal of this job? Is this no longer needed? Can it be removed?

Reply:
This sidekiq job should have been removed in a previous PR (https://github.com/department-of-veterans-affairs/vets-api/pull/5862) that removed the `AppsApi::DirectoryLoader` class in a refactor. It slipped through the cracks and does nothing currently so its removal got added to this pr.